spotify_player/src/config/mod.rs

```diff
@@ -348,6 +349,7 @@ impl Default for LayoutConfig {
             },
             playback_window_position: Position::Top,
             playback_window_height: 6,
+            title_height: 1,
```

Comment:
This PR requires #482 because the config option is implemented in the new Layout section of the config, which is added in that PR.

But considering the extensive changes made/requested to t#482, it might be better to close this PR  until that one is merged.

Reply:
I hope this answers your question.